test/unit/util.test.js

```diff
@@ -2,8 +2,8 @@
 require = require('esm')(
   module /* , options */
 ); /* eslint-disable-line no-global-assign */
-const { resolvePath } = require('../../src/core/router/util');
 const { expect } = require('chai');
+const { resolvePath } = require('../../src/core/router/util');
```

Comment:
Let's avoid changes that don't have meaning related to the current pull request. Thoughts @anikethsaha ?

Reply:
yes 👍 
this should be removed

Reply:
this was changed by cli tool, not by manual